docs/compatibility.md

```diff
@@ -377,13 +377,6 @@ date. Typically, one that overflowed.
 
 ### CSV Floating Point
 
-The CSV parser is not able to parse `NaN` values.  These are
-likely to be turned into null values, as described in this
-[issue](https://github.com/NVIDIA/spark-rapids/issues/125).
-
-Some floating-point values also appear to overflow but do not for the CPU as described in this
-[issue](https://github.com/NVIDIA/spark-rapids/issues/124).
-
 Any number that overflows will not be turned into a null value.
```

Comment:
I've removed this and added the disclaimer that we have the same issues as casting from float to string. I have enabled tests for overflow but had to remove the NaN / Inf values from `simple_floats_values.csv` but I think that is ok since we already have those values in `nans_and_inf.csv` which is covered by the same test but currently XFAILs due to known issues.